code/src/main/java/frc/robot/subsystems/SwerveDrivetrain.java

```diff
@@ -73,30 +67,30 @@ public SwerveDrivetrain(final ILogger logger, final RobotConfig config) {
         m_frontLeftTurn = new CANSparkMax(config.Drivetrain.frontLeftTurnMotorPort, MotorType.kBrushless);
         m_frontLeftEncoder = new AbsoluteEncoder(config.Drivetrain.frontLeftAbsoluteEncoder, 2.254991, true);
 
-        m_frontLeft = new SwerveWheel(m_frontLeftDrive, m_frontLeftTurn, config.Drivetrain.swerveX,
-                config.Drivetrain.swerveY, m_frontLeftEncoder, "Left front");
+        m_frontLeft = new SwerveWheel(m_frontLeftDrive, m_frontLeftTurn, config.Physical.widthInMeters / 2,
+                config.Physical.lengthInMeters / 2, m_frontLeftEncoder, "Left front");
```

Comment:
this should not be needed. we accounted for this when we applied the numbers originally